mu-plugins/central-hub/tests/phpunit/fixtures/test-cpt-config.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ *  Test data for the ConfigStore API.
+ *
+ * @package    spiralWebDb\centralHub\Tests\Fixtures
+ * @since      1.0.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Fixtures;
+
+return  [
+	'foo' => [
+		'aaa' => 'bbb',
+		'ccc' => 'ddd'
+	]
+];
```

Comment:
@rgadon107 Make sure there's an empty line at the end of PHP files.

Reply:
Added new line at end of file.

Reply:
Added new line at end of file per code review.